providers/attio/metadata.go

```diff
@@ -0,0 +1,77 @@
+// nolint
```

Comment:
Hey @sanjay-a01 ,Is there a reason for disabling linter in these files: metadata, ObjectNames, read, write?

Reply:
Yeah, I faced the lint issue while raising the PR so only I have disabled the linter.

Reply:
Yeah, I dont think it should be disabled entirely. Some things are to be addressed and some should be ignored line by line. Otherwise, if more inconsistencies are added the linter would not report them, which is not good.